types/sanctuary/sanctuary-tests.ts

```diff
@@ -1,10 +1,73 @@
 /// <reference types="node"/>
-import * as assert from "assert";
-import { create, env } from "sanctuary";
+import * as assert from 'assert';
+import { create, env } from 'sanctuary';
 
-const checkTypes = process.env["NODE_ENV"] !== "production";
-const S = create({checkTypes, env});
+const checkTypes = process.env['NODE_ENV'] !== 'production';
+const S = create({ checkTypes, env });
 
-assert.equal(S.map(S.concat('@'))(['foo', 'bar', 'baz']), ["@foo", "@bar", "@baz"]);
+assert.equal(S.map(S.concat('@'))(['foo', 'bar', 'baz']), ['@foo', '@bar', '@baz']);
 assert.equal(S.reduce(S.add)(0)([1, 2, 3, 4, 5]), 15);
-assert.equal(S.flip(S.concat)('foo')('bar'), "barfoo");
+assert.equal(S.flip(S.concat)('foo')('bar'), 'barfoo');
```

Comment:
Can we remove those lines? They are testing lib behavior, not typing

Reply:
Good idea. :thumbsup: